dev-server/mock-routes.js

```diff
@@ -34,4 +36,60 @@ app.get('/familie-ba-sak/api/behandling/1/vedtak-html', (req, res) => {
     );
 });
 
+app.post('/familie-ba-sak/api/behandling/opprett', (req, res) => {
```

Comment:
Why do we need this? Can't we just use the json files (create some more)?

Reply:
Yes, and I will make the json files